pom.xml

```diff
@@ -40,6 +40,7 @@
     <skip.integration.tests>true</skip.integration.tests>
   </properties>
   <build>
+  <pluginManagement>
```

Comment:
try to remove this pluginManagement lines, otherwise the build must fail (see travis CI)